live-site/components/welcome-section/team-race/TeamRace.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { H3, P } from '../../../../shared-ui/style/typography';
+import { TeamProps, TeamRaceProps } from '../../../lib/types';
+import {
+  StyledRaceContainer,
+  StyledRaceContent
+} from '../WelcomeSection.styles';
+import Team from './Team';
+
+const TeamRace: React.FC<TeamRaceProps> = ({ teams }) => {
+  const orderTeams = (teams: TeamProps[]): TeamProps[] => {
+    return teams.sort((team1: TeamProps, team2: TeamProps) =>
+      team1.points > team2.points ? -1 : -1
+    );
+  };
+
+  return (
+    <>
+      <StyledRaceContainer>
+        <StyledRaceContent>
+          <H3>DesertRace</H3>
+          <P>{`Earn points for yourself and your cabin by participating in events! Let’s see which cabin wins the 2023 Desert Race!`}</P>
```

Comment:
Why does this need to be in {``}? Can't we just put the text as is here?

Reply:
it doesn't go into a new line. unless i use <br> which is generally discouraged.